### PR TITLE
Auth user add product component

### PR DESCRIPTION
**Pull request by danieltichiyama**
This PR does not contain any error handling or form validation, which will need to be added later.

It removes the manual id fields for all of the seeds, which was causing errors when trying to add new items to the database, and also adds the change to the Dockerfile to correct an bug in the docker container that prevented the server from refreshing after a new save.

This PR adds the AddProduct component.  From the base app, you should:
- [ ] see an Add Item component
- [ ] be able to input values into the fields
- [ ] hit submit and add a new product to the database
- [ ] a message "Item successfully added" should appear.

### CODE REVIEW

File: server/database/seeds/3_images.js [hunk 1]
```diff
@@ -6,67 +6,56 @@ exports.seed = function(knex) {
...
           item_id: 8
         },
         {
-          id: 10,
           url:
             "https://myfixituplife.com/wp-content/uploads/2014/01/hoarding.jpg",
           item_id: 9
         },
         {
-          id: 11,
```
Comment (rianmueller) [inline server/database/seeds/3_images.js]: You shouldn't have to remove all the ids in the seed data unless it's breaking your workflow right now, just wait to merge the branch where I've already done that.

Comment (danieltichiyama) [reply server/database/seeds/3_images.js]: I did that before we discussed them.  I'm going to leave them in, if that's alright. I don't think they will cause any merge conflicts, will they?

File: src/containers/HabitList/HabitList.jsx [hunk 2]
```diff
@@ -1,4 +1,8 @@
 import React, { Component } from "react";
+<<<<<<< HEAD
+import Habit from "../../components/Habit";
```
Comment (jkhirano) [inline src/containers/HabitList/HabitList.jsx]: Need to delete "import Habit..."

**Review by andrewobx — APPROVED** [review]
Is gud code.

File: server/database/seeds/3_images.js (hunk 1 above)
Comment (rianmueller) [reply server/database/seeds/3_images.js]: Leaving them in won't cause any merge conflicts. The changes will be made on dev when add-auth is merged to it, and they should propagate to your local branch when you update from dev.

**Pull request merged**